From fc70c7fa9b342c08c9e80253c45bf5d7e33b8d74 Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Mon, 11 Oct 2021 18:25:37 +0530
Subject: [PATCH] Remove DisableCameraReceiver feature to avoid icon not found
 issue

- Camera2 app has a feature to disable the camera launcher
  activity if there is no Camera detected in the first
  CameraProvider call during the boot, but this is not
  applicable for cic cloud remote camera. Hence disable
  this feature in Camera2 app to avoid camera issues in
  the run time.
- Note: This feature doesn't have dependency on any
  camera use cases.

Tracked_on: OAM-99724
Change-Id: Ifa5a56afb502f86156afcce4c967e7ce66c7efef
Signed-off-by: Muhammad Aksar <muhammad.aksar@intel.com>
---
 AndroidManifest.xml | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 0e21564dc..88b1a76de 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -154,11 +154,6 @@
             android:configChanges="keyboardHidden|orientation|screenSize">
         </activity>
 
-        <receiver android:name="com.android.camera.DisableCameraReceiver">
-            <intent-filter>
-                <action android:name="android.intent.action.BOOT_COMPLETED" />
-            </intent-filter>
-        </receiver>
     </application>
 
 </manifest>
-- 
2.33.0

